tests/next_tests/otf_tests/test_pybind_integration_test.py

```diff
@@ -0,0 +1,49 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2023, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+import numpy as np
+
+from gt4py.next import common
+from gt4py.next.ffront.decorator import field_operator
+from gt4py.next.ffront.fbuiltins import int32
+from gt4py.next.iterator.embedded import np_as_located_field
+from gt4py.next.program_processors.runners.gtfn_cpu import run_gtfn
```

Comment:
Direct object imports go against our coding conventions.

Reply:
In this case I actively decided against them: I believe they should all be imported from a unique place (so I followed existing, but not-compliant practices). I can change it here, or we clean up all tests at once.

Reply:
I'll change it, as I anyway touch the code for adding the comment.